tests/Js/JavascriptTest.php

```diff
@@ -39,4 +39,13 @@ public function testIssue225()
 
         $this->assertContains('OH AIH!', $this->getSession()->getPage()->getText());
     }
+
+    public function testAutocomplete()
+    {
+        $this->getSession()->visit($this->pathTo('/autocomplete.html'));
+        $this->getSession()->getPage()->fillField('autocomplete', 'arc');
+        $this->getSession()->wait(5000, '$("ul.ui-autocomplete li").is(":visible") === true');
+
+        $this->assertContains('Arctic fox', $this->getSession()->getPage()->getText());
```

Comment:
When you fill `arc` into the input, then auto-complete results will show, but current field value won't be set to 1st match. Or does work like that?

Reply:
I used the jQuery UI Autocomplete library which doesn't automatically fill up the field as you type. It shows the results in a dropdown, you can then use the arrow down / arrow up keys to select the right value, and then pressing enter will put the value in the field.

The test is now just checking that the autocomplete results show up in the dropdown. If you want I can also add coverage for selecting the results with the arrow keys.